tests/cucumber/features/step_definitions/organizations_steps.coffee

```diff
@@ -40,7 +40,7 @@ do ->
 
     @Then /^I should be on the "([^"]*)" detail page$/, (name, callback) ->
       @browser
-        .waitForVisible('.organization-detail', assert.ifError)
+        .waitForVisible('.organization-detail', 30000, assert.ifError)
```

Comment:
Are we able to detect the presence of environment variables in that coffeescript? Perhaps the timeout value can be generated based on the presence or absence of the $CIRCLECI variable?
https://circleci.com/docs/environment